src/include/sof/lib/alloc.h

```diff
@@ -67,6 +67,8 @@ enum mem_zone {
 
 /** \brief Indicates that original content should not be copied by realloc. */
 #define SOF_MEM_FLAG_NO_COPY	BIT(1)
+/** \brief Indicates that if we should return uncached address. */
+#define SOF_MEM_FLAG_COHERENT  BIT(2)
```

Comment:
This should be in CAPS i.e ```SOF_MEM_CAPS_COHERENT```

Reply:
not following @lgirdwood do you want this to be part of the zone capabilities, or part of the requirement from the caller?

Reply:
Sorry, I mean.
```c
md = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_COHERENT, sizeof(*md));
```

Reply:
then add this SOF_MEM_CAPS_COHERENT CAPS to which Zone? Only to the _BUFFER?

Reply:
all zones, it should be a generic capability. the zone will return an uncached ptr if set otherwise an cached ptr.

Reply:
ping. CI looking good so far.

Reply:
> ping. CI looking good so far.

Let me ping and talk to you today about the details @lgirdwood

Reply:
ok, I think with part 4 & 5 we then align more closely with Zephyr which is great.